asd-aituber/apps/web/lib/session-manager.ts

```
@@ -0,0 +1,173 @@
+
+  /**
+   * ユーザーIDを設定（将来の認証システム用）
+   */
+  static setUserId(userId: string): void {
+    const session = this.loadSession()
+    if (!session) return
+    
+    session.userId = userId
+    this.saveSession(session)
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)